planner/core/point_get_plan.go

```diff
@@ -362,6 +363,10 @@ func TryFastPlan(ctx sessionctx.Context, node ast.Node) Plan {
 	ctx.GetSessionVars().PlanColumnID = 0
 	switch x := node.(type) {
 	case *ast.SelectStmt:
+		if ctx.GetSessionVars().SelectLimit != math2.MaxUint64 {
+			ctx.GetSessionVars().StmtCtx.AppendWarning(errors.New("sql_select_limit is set, so we will not get fast plan."))
```

Comment:
```suggestion
			ctx.GetSessionVars().StmtCtx.AppendWarning(errors.New("sql_select_limit is set, so point get plan is not activated."))
```

Reply:
Could we give this warning only if the point get plan is valid but canceled by `sql_select_limit`?

Reply:
ok